keycloak_client.go

```diff
@@ -421,10 +422,12 @@ func str(s string) *string {
 
 func whitelistErrors(statusCode int, message string) error {
 	// whitelist errors from Keycloak
+	reg := regexp.MustCompile("invalidPassword[a-zA-Z]*Message")
 
-	switch message {
-	//POST account/credentials/password with error message "invalidPasswordExistingMessage"
-	case "invalidPasswordExistingMessage":
+	switch {
```

Comment:
if / else would be easier

Reply:
True, but we try to prepare the ground for all possible error messages. In the future more errors will be whitelisted (or we start with this assumption)